packages/components/src/2016-12-01/button-state.test.js

```diff
@@ -16,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 */
-const chai = require('chai');
+const chai = require('chai'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
It's named as a dev dependency. The linter really wants me to add it as a client dependency. Our clients shouldn't need to download chai to use a component.